ad-click/ad-click-impl/src/main/java/com/duckduckgo/adclick/impl/DataRemovalAdClickWorker.kt

```diff
@@ -36,6 +36,7 @@ import timber.log.Timber
 import java.util.concurrent.TimeUnit
 import javax.inject.Inject
 
+@Suppress("NoHardcodedCoroutineDispatcher")
```

Comment:
Should be able to inject `DispatcherProvider` like in `AppTrackerExclusionListUpdateWorker`

Reply:
Thanks. I'd shied away from making some changes to de-risk, but on 2nd look it should be safe enough.

Done